ToolsTest/Volodymyr/xUnit.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Tools;
+
+using Xunit;
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
```

Comment:
The same as in comment for the `nUnit.cs` file.
You are using the Assert class from the MSTest but expected is NUnit.
To be sure that you are using the proper class just remove it (as here it is needless), run the tests from this file, and check that they are success.